routes/duties.js

```diff
@@ -0,0 +1,48 @@
+const serverUrl = require("../config").serverUrl;
+const utils = require("../utility/utility");
+
+function handleRequest(method, url, data) {
+  const router = require("../start").server.getRouter();
+  switch (method) {
+    case "POST":
+      router.routeToNewDutie(data);
+      break;
+    case "GET":
+      const parsedUrl = new URL(serverUrl + url);
```

Comment:
to manu branches in this if, logic hard to follow. make it switch case or reduce amount needed. 

if you need more than an if and an else, then you generally have too complicated a logic presented